src/OpenRiaServices.Tools/Framework/CreateOpenRiaClientFilesTask.cs

```diff
@@ -770,7 +770,11 @@ private bool GenerateClientProxiesOutOfProcess(string generatedFileName, ClientC
         {
             // Call the console app from here if Net 6.0 or greater
             string path = Path.Combine(Path.GetDirectoryName(typeof(CreateOpenRiaClientFilesTask).Assembly.Location),
+#if NET8_0
+                "../net8.0/OpenRiaServices.Tools.CodeGenTask.exe");
```

Comment:
We should do some manual test and se if we can stay with shipping .net6 a little while longer and rely on roll-forward behaviour.

Otherwise we need to make this switch depending on what framework the server project targets.
- As son as we have released aspnetcore hosting with only net8 target we can consider just switching to hardcoded .net8 here.
- As long as we test it once on a net6 (and maybe net7) project we could change CodeGenTask.exe to be .NET 8 only

Reply:
Seems to work fine to use net6